product_links/README.rst

```diff
@@ -0,0 +1,36 @@
+a module adds links between products
+===========================================
```

Comment:
what do you mean "should be of the sale length" ?  the same length with the sale module ?